pkg/pillar/cast/cast.go

```diff
@@ -401,7 +401,7 @@ func CastUuidToNum(in interface{}) types.UuidToNum {
 	return output
 }
 
-// ZbootConfig converts/casts from an interface to a ZbootConfig
+// ZbootConfig : casts interface to ZbootConfig
```

Comment:
I note that ZbootConfig is only about TestComplete. At some point in time it might make sense to rename it for clarity.